tests/models/cpmant/test_modeling_cpmant.py

```diff
@@ -0,0 +1,257 @@
+# coding=utf-8
+# Copyright 2022 The OpenBMB Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch CPMAnt model. """
+
+import unittest
+
+from transformers.testing_utils import is_torch_available, require_torch, slow
+
+from ...generation.test_utils import torch_device
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import ModelTesterMixin, ids_tensor
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        CPMAntConfig,
+        CPMAntForCausalLM,
+        CPMAntModel,
+        CPMAntTokenizer,
+    )
+
+
+class CPMAntConfigTester(ConfigTester):
+    def create_and_test_config_common_properties(self):
+        config = self.config_class(**self.inputs_dict)
+        self.parent.assertTrue(hasattr(config, "dim_model"))
+        self.parent.assertTrue(hasattr(config, "num_heads"))
+        self.parent.assertTrue(hasattr(config, "dim_head"))
```

Comment:
Can you explain why you need to ovveride the `ConfigTester` here? Check: https://github.com/huggingface/transformers/blob/dcec3277cdbbeafb6107e78f87d701111196c78a/tests/models/opt/test_modeling_opt.py#L212

Reply:
Because our original model config has some different names. Actually it's not necessary, and I have renamed the arguments.